lib/stream.js

```diff
@@ -83,6 +84,22 @@ module.exports = class UDXStream extends streamx.Duplex {
     return this._view[binding.offsetof_udx_stream_t_inflight >> 2]
   }
 
+  get bytesOut () {
+    return Number(this._view64[binding.offsetof_udx_stream_t_bytes_out >> 3])
```

Comment:
Using BigInts is annoying, because they can't be compared with Numbers (`0 !== 0n), so I think it's better to cast to Number before returning them to Javascript-land.

Note: in theory, casting a BigInt to Number can lose precision, but the first number which loses precision is `Number.MAX_SAFE_INTEGER === 9007199254740991`, which is over 9000 Terabytes, so it seems fine (it should not fail catastrophically at that point, just introduce rounding errors).